scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -239,7 +239,7 @@ object TreeSyntax {
           case p: Pat.Wildcard => unreachable
```

Comment:
wouldn't this and Typed below also need to be fixed? @dos65

Reply:
I think there is no need. `Pat.Wildcard` can't be a parent of Term.Name.
What about `Pat.Typed` - I haven't come up with any case when it might be reachable here.